### ISSUE TITLE
OMERO.server should log to stdout/stderr for docker logs

### ISSUE THREAD

**Issue by manics** [openmicroscopy/omero-server-docker#9]
OMERO.server writes its logs to files under /opt/omero/web/OMERO.server/var/log/. It should write to stdout/stderr instead so they can be seen by docker logs

**Comment by joshmoore** [comment]
All of them? If so, prefix with `[Blitz-0]`, `[PixelData-0]`, etc?

**Comment by manics** [comment]
Yes, unless they're run as separate containers. Presumably this is straightforward for the someone who understands Java logback? IIRC the Python servers are more complicated due to some mangling of file descriptors.

**Comment by joshmoore** [comment]
Yeah, I don't think Java is much of an issue. Probably one small-ish logback file per server. For Python, we likely need a code change to allow modifying https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/util/__init__.py#L79
